lib/logstash/plugin_mixins/jdbc.rb

```diff
@@ -76,7 +99,7 @@ def prepare_jdbc_connection
                 end
       raise LogStash::ConfigurationError, "#{e}. #{message}"
     end
-    @database = Sequel.connect(@jdbc_connection_string, :user=> @jdbc_user, :password=>  @jdbc_password.nil? ? nil : @jdbc_password.value)
+    @database = jdbc_connect()
```

Comment:
updated to log error when failed to connect because of timeout related reasons, as well as others.